SoftLayer/CLI/hardware/create.py

```diff
@@ -129,11 +133,15 @@ def cli(env, **args):
 
         result = mgr.place_order(**order)
 
-        table = formatting.KeyValueTable(['name', 'value'])
-        table.align['name'] = 'r'
-        table.align['value'] = 'l'
-        table.add_row(['id', result['orderId']])
-        table.add_row(['created', result['orderDate']])
-        output = table
+        if args['output_json']:
+            env.fout(json.dumps({'orderId': result['orderId']}))
```

Comment:
I can add created.

The id thing is kind of a headache, the SL API makes use of 2 different IDs: "orderId" and "hardwareId" in different contexts/endpoints but calls them both "id" even though the can kinda refer to the same hardware. I opted to make the orderId explicitly labeled so, so it is slightly less confusing.